server/utils/courseUtils.test.ts

```diff
@@ -1,13 +1,28 @@
-import courseWithPrerequisiteSummaryListRows from './courseUtils'
-import { courseFactory } from '../testutils/factories'
+import courseUi from './courseUtils'
+import { courseAudienceFactory, courseFactory } from '../testutils/factories'
 
 describe('courseUtils', () => {
-  describe('courseWithPrerequisiteSummaryListRows', () => {
-    it('adds prerequisite summary list rows to a course', () => {
-      const course = courseFactory.build()
+  describe('courseUi', () => {
```

Comment:
can we call this `presentCourse` or `decorateCourse` and have it return a `presentedCourse` or `decoratedCourse` - it's essentially following the [Decorator design pattern](https://refactoring.guru/design-patterns/decorator) - I think the term `UI` is overloaded by being part of the name of our application!